convex/_generated/clientConfig.d.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
```

Comment:
Should generated files be tracked in Git?

Reply:
Yeah, it's nice for the project to still compile even if someone isn't running convex locally.